src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'ADD_BOOK';
```

Comment:
- [ ] please kindly ensure that your action typs follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), for `add book`  it will be something like:       const ADD_BOOK = bookstore/books/ADD_BOOK